test/specs/controller.bubble.tests.js

```diff
@@ -1,4 +1,6 @@
 describe('Chart.controllers.bubble', function() {
+	describe('auto', jasmine.specsFromFixtures('controller.bubble'));
```

Comment:
Is there a reason we need this `describe` block but we don't need anything for the `element.point` tests?

Reply:
There is the same [describe('auto') for `element.point`](/chartjs/Chart.js/pull/5629/files#diff-a707c6497c455b149037a59c0d7030c3R2). It's used to group all `it()` generated for each `.json` in the fixture folder under the same "auto" label, so I think it's better to have it:

![image](https://user-images.githubusercontent.com/3874900/43367197-afdc2802-9349-11e8-9848-7e0cf016291a.png)